core/src/main/scala/kafka/consumer/FetchRequestAndResponseStats.scala

```diff
@@ -34,6 +34,7 @@ class FetchRequestAndResponseMetrics(metricId: ClientIdBroker) extends KafkaMetr
 
   val requestTimer = new KafkaTimer(newTimer("FetchRequestRateAndTimeMs", TimeUnit.MILLISECONDS, TimeUnit.SECONDS, tags))
   val requestSizeHist = newHistogram("FetchResponseSize", biased = true, tags)
+  val throttleTimeHist = newHistogram("FetchRequestThrottleTimeMs", biased = true, tags)
```

Comment:
Sure. I have used timer instead of histogram for the metrics. Please see the updated patch.